source/release/4.4.6/index.md

```diff
@@ -703,7 +682,7 @@ In order to customize or disable the above default settings it is recommended to
 
 #### Contributors
 
-47 people contributed to this release:
+52 people contributed to this release:
```

Comment:
@sandrobonazzola What about the tech writers?

Reply:
Would be happy to add them :-) Contributors list here is auto-generated from commit messages, adding manually

Reply:
Gotcha.